### PR TITLE
Support alertmanager Grafana datasource

### PR DESCRIPTION
**Pull request by jgosmann**
##### SUMMARY
This adds the "alertmanager" datasource. [Apparently added in Grafana 9.2.](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v9-2/#configure-external-alertmanagers-as-data-sources)

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

grafana_datasource plugin

### CODE REVIEW

**Comment by rrey** [comment]
Hi @jgosmann, 

Please add integration tests and a changelog fragment to ha e CI pass.

thx 🙏🏻 !

**Comment by BenjaminSchubert** [comment]
Hey, I'd be interested in being able to do this too. I see the tests were added but the PR has become stale. Is there any hope of resurrecting this PR @jgosmann or one of the maintainers? Or would it be best to create a new one?

**Comment by jgosmann** [comment]
I don't think there's anything I can do at the moment. Still waiting for a review. 😅

**Review by BenjaminSchubert — COMMENTED** [review]
(Not a maintainer, mainly trying to help :) )

The code seems consistent with the other implementations of data sources, I also tried it locally and it did work as I expected. There are conflicts that will need rebasing though

**Comment by Nemental** [comment]
Hey @jgosmann,
I've resolved the conflicts and made a few other small changes.
Tomorrow I'll do the final review and take a look over the tests, it's just too late for that now.
Many thanks for the PR and your contribution!

File: plugins/modules/grafana_datasource.py
Comment (rndmh3ro) [inline plugins/modules/grafana_datasource.py:243]:
```suggestion
    - Whether Grafana should send alerts to this alertmanager.
```

**Review by Nemental — APPROVED** [review]
lgtm! thanks @jgosmann

**Pull request merged**
